### ISSUE TITLE
Error when using the function

### ISSUE THREAD

**Issue by SuX97** [interactiveaudiolab/emphases#40]
Hi authors!

I'm testing the function to find emphased words, an error occor:

Usage:
```
python -m emphases --text_files ../demo_data/audio/avatar_jenny_excited.txt  --audio_files ../demo_data/audio/avatar_jenny_excited.wav
```

Log
```
  ERROR [+8522]  LatFromPaths: Align have dur<=0
 FATAL ERROR - Terminating program HVite
multiprocessing.pool.RemoteTraceback:
"""
Traceback (most recent call last):
  File "/Users/agi00072/anaconda3/envs/a2f/lib/python3.8/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/Users/agi00072/anaconda3/envs/a2f/lib/python3.8/multiprocessing/pool.py", line 51, in starmapstar
    return list(itertools.starmap(args[0], args[1]))
  File "/Users/agi00072/anaconda3/envs/a2f/lib/python3.8/site-packages/pyfoal/baselines/p2fa.py", line 61, in from_file_to_file
    from_file(text_file, audio_file).save(output_file)
  File "/Users/agi00072/anaconda3/envs/a2f/lib/python3.8/site-packages/pyfoal/baselines/p2fa.py", line 56, in from_file
    return from_text_and_audio(text, audio, sample_rate)
  File "/Users/agi00072/anaconda3/envs/a2f/lib/python3.8/site-packages/pyfoal/baselines/p2fa.py", line 44, in from_text_and_audio
    return from_text_and_audio.aligner(text, audio, duration)
  File "/Users/agi00072/anaconda3/envs/a2f/lib/python3.8/site-packages/pyfoal/baselines/p2fa.py", line 132, in __call__
    alignment = self.correct_alignment(alignment_file, duration)
  File "/Users/agi00072/anaconda3/envs/a2f/lib/python3.8/site-packages/pyfoal/baselines/p2fa.py", line 150, in correct_alignment
    durations[0] += .0125
IndexError: list index out of range
"""
```

The file I used is attached, hope you guys can help!

[avatar_jenny_excited.wav.zip](https://github.com/interactiveaudiolab/emphases/files/14359960/avatar_jenny_excited.wav.zip)
[avatar_jenny_excited.txt](https://github.com/interactiveaudiolab/emphases/files/14359979/avatar_jenny_excited.txt)

**Comment by maxrmorrison** [comment]
Thanks for opening an issue! Our current hypothesis is that P2FA (the forced aligner) is failing when there are leading directories. We're working on a fix. For now, you might be able to fulfill your needs by running the command from `demo_data/audio`. @NathanPruyne mentioned he was able to get your files working properly when there were no leading directories in the path name.

**Comment by maxrmorrison** [comment]
Fixed in version 0.0.2!
```
(emphases) max@malleus:~/emphases$ python -m emphases --text_files ../demo_data/audio/avatar_jenny_excited.txt  --audio_files ../demo_data/audio/avatar_jenny_excited.wav
emphases: 100%|█████████████████████████████████████████████| 1/1 [00:00<00:00,  2.53it/s]
(emphases) max@malleus:~/emphases$
```

**Comment by liangzhaohui** [comment]
I am in the same situation and using emphases v0.0.2 + pyfoal v1.0.1. How did author solve this problem?

root@vultr:~/emphases_backup# python3 -m emphases --text_files avatar_jenny_excited.txt --audio_files avatar_jenny_excited.wav
  ERROR [+8522]  LatFromPaths: Align have dur<=0 
 FATAL ERROR - Terminating program HVite
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/usr/lib/python3.10/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/usr/lib/python3.10/multiprocessing/pool.py", line 51, in starmapstar
    return list(itertools.starmap(args[0], args[1]))
  File "/usr/local/lib/python3.10/dist-packages/pyfoal/baselines/p2fa.py", line 61, in from_file_to_file
    from_file(text_file, audio_file).save(output_file)
  File "/usr/local/lib/python3.10/dist-packages/pyfoal/baselines/p2fa.py", line 56, in from_file
    return from_text_and_audio(text, audio, sample_rate)
  File "/usr/local/lib/python3.10/dist-packages/pyfoal/baselines/p2fa.py", line 44, in from_text_and_audio
    return from_text_and_audio.aligner(text, audio, duration)
  File "/usr/local/lib/python3.10/dist-packages/pyfoal/baselines/p2fa.py", line 132, in __call__
    alignment = self.correct_alignment(alignment_file, duration)
  File "/usr/local/lib/python3.10/dist-packages/pyfoal/baselines/p2fa.py", line 150, in correct_alignment
    durations[0] += .0125
IndexError: list index out of range
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/root/emphases_backup/emphases/__main__.py", line 50, in <module>
    emphases.from_files_to_files(**vars(parse_args()))
  File "/root/emphases_backup/emphases/core.py", line 153, in from_files_to_files
    pyfoal.from_files_to_files(
  File "/usr/local/lib/python3.10/dist-packages/pyfoal/core.py", line 216, in from_files_to_files
    pyfoal.baselines.p2fa.from_files_to_files(
  File "/usr/local/lib/python3.10/dist-packages/pyfoal/baselines/p2fa.py", line 78, in from_files_to_files
    pool.starmap(align_fn, iterator)
  File "/usr/lib/python3.10/multiprocessing/pool.py", line 375, in starmap
    return self._map_async(func, iterable, starmapstar, chunksize).get()
  File "/usr/lib/python3.10/multiprocessing/pool.py", line 774, in get
    raise self._value
IndexError: list index out of range
root@vultr:~/emphases_backup# emphases --version
emphases: command not found
root@vultr:~/emphases_backup# pip list
Package                  Version
------------------------ ----------------
absl-py                  2.1.0
acme                     1.21.0
appdirs                  1.4.4
apprise                  1.8.0
attrs                    21.2.0
audioread                3.0.1
Automat                  20.2.0
Babel                    2.8.0
bcrypt                   3.2.0
bibtexparser             2.0.0b7
blinker                  1.4
boto3                    1.34.120
botocore                 1.34.120
certbot                  1.21.0
certbot-nginx            1.21.0
certifi                  2020.6.20
cffi                     1.16.0
chardet                  4.0.0
click                    8.0.3
clldutils                3.22.2
cloud-init               23.4.3
colorama                 0.4.4
colorlog                 6.8.2
command-not-found        0.3
ConfigArgParse           1.5.3
configobj                5.0.6
constantly               15.1.0
contourpy                1.2.1
cryptography             3.4.8
csvw                     3.3.0
cycler                   0.12.1
dbus-python              1.2.18
decorator                5.1.1
Distance                 0.1.3
distro                   1.7.0
distro-info              1.1+ubuntu0.2
dlinfo                   1.2.1
emphases                 0.0.2
filelock                 3.14.0
fonttools                4.53.0
fsspec                   2024.6.0
g2p-en                   2.1.0
GPUtil                   1.4.0
grpcio                   1.64.1
heroku3                  5.2.1
httplib2                 0.20.2
huggingface-hub          0.23.3
hyperlink                21.0.0
idna                     3.3
importlib-metadata       4.6.4
incremental              21.3.0
inflect                  7.2.1
isodate                  0.6.1
jeepney                  0.7.1
Jinja2                   3.0.3
jmespath                 1.0.1
joblib                   1.4.2
josepy                   1.10.0
jsonpatch                1.32
jsonpointer              2.0
jsonschema               3.2.0
keyring                  23.5.0
kiwisolver               1.4.5
language-tags            1.2.0
launchpadlib             1.10.16
lazr.restfulclient       0.14.4
lazr.uri                 1.0.6
lazy_loader              0.4
librosa                  0.10.2.post1
llvmlite                 0.42.0
lxml                     5.2.2
Markdown                 3.6
markdown-it-py           3.0.0
MarkupSafe               2.1.5
matplotlib               3.9.0
mdurl                    0.1.2
more-itertools           8.10.0
mpmath                   1.3.0
msgpack                  1.0.8
mysql-connector-python   8.4.0
netifaces                0.11.0
networkx                 3.3
nltk                     3.8.1
numba                    0.59.1
numpy                    1.26.4
nvidia-cublas-cu11       11.10.3.66
nvidia-cublas-cu12       12.1.3.1
nvidia-cuda-cupti-cu12   12.1.105
nvidia-cuda-nvrtc-cu11   11.7.99
nvidia-cuda-nvrtc-cu12   12.1.105
nvidia-cuda-runtime-cu11 11.7.99
nvidia-cuda-runtime-cu12 12.1.105
nvidia-cudnn-cu11        8.5.0.96
nvidia-cudnn-cu12        8.9.2.26
nvidia-cufft-cu12        11.0.2.54
nvidia-curand-cu12       10.3.2.106
nvidia-cusolver-cu12     11.4.5.107
nvidia-cusparse-cu12     12.1.0.106
nvidia-nccl-cu12         2.20.5
nvidia-nvjitlink-cu12    12.5.40
nvidia-nvtx-cu12         12.1.105
oauthlib                 3.2.0
packaging                24.0
parsedatetime            2.6
penn                     0.0.13
pexpect                  4.8.0
phonemizer               3.2.1
pillow                   10.3.0
pip                      22.0.2
platformdirs             4.2.2
pooch                    1.8.1
protobuf                 5.27.0
psutil                   5.9.8
ptyprocess               0.7.0
pyasn1                   0.4.8
pyasn1-modules           0.2.1
pycparser                2.22
pycwt                    0.4.0b0
pyfoal                   1.0.1
Pygments                 2.18.0
PyGObject                3.42.1
PyHamcrest               2.0.2
PyICU                    2.8.1
PyJWT                    2.3.0
pylatexenc               2.10
pyOpenSSL                21.0.0
pypar                    0.0.6
pyparsing                2.4.7
pyRFC3339                1.1
pyrsistent               0.18.1
pyserial                 3.5
python-apt               2.4.0+ubuntu3
python-dateutil          2.9.0.post0
python-debian            0.1.43+ubuntu1.1
python-dotenv            1.0.1
python-magic             0.4.24
pytz                     2022.1
PyYAML                   6.0.1
rdflib                   7.0.0
regex                    2024.5.15
requests                 2.25.1
requests-oauthlib        2.0.0
requests-toolbelt        0.9.1
resampy                  0.4.3
reseval                  0.1.6
rfc3986                  1.5.0
rich                     13.7.1
s3transfer               0.10.1
scikit-learn             1.5.0
scipy                    1.13.1
SecretStorage            3.3.1
segments                 2.2.1
service-identity         18.1.0
setuptools               59.6.0
six                      1.16.0
sos                      4.5.6
soundfile                0.12.1
soxr                     0.3.7
ssh-import-id            5.11
sympy                    1.12.1
systemd-python           234
tabulate                 0.9.0
tensorboard              2.16.2
tensorboard-data-server  0.7.2
threadpoolctl            3.5.0
torch                    1.13.1
torchaudio               0.13.1
torchutil                0.0.13
tqdm                     4.66.4
triton                   2.3.1
Twisted                  22.1.0
typeguard                4.3.0
typing_extensions        4.12.1
ubuntu-advantage-tools   8001
ubuntu-drivers-common    0.0.0
ufw                      0.36.1
unattended-upgrades      0.1
uritemplate              4.1.1
urllib3                  1.26.5
wadllib                  1.3.6
Werkzeug                 3.0.3
wheel                    0.37.1
xkit                     0.0.0
xmltodict                0.13.0
yapecs                   0.0.8
zipp                     1.0.0
zope.component           4.3.0
zope.event               4.4
zope.hookable            5.1.0
zope.interface           5.4.0
